### PR TITLE
Chapter 3 - Static Pages

### PR DESCRIPTION
**Pull request by chienpd-0036**
Chapter 3 - Static Pages + I18n
Screen Shot 2019-06-28 at 12 32 30

### CODE REVIEW

**Comment by chienpd-0036** [comment]
ready

File: app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% provide :title, t("static_pages.about.title") %>
+<h1> <%= t "static_pages.about.title" %> </h1>
+<p>
+  <%= t "static_pages.about.sentence_1" %> <a href="https://www.railstutorial.org/"><em><%= t "static_pages.about.link_1" %></em></a> <%= t "static_pages.about.sentence_2" %>
+  <a href="https://www.railstutorial.org/book"><%= t "static_pages.about.link_2" %></a> <%= t "static_pages.about.sentence_3" %>
```
Comment (thanhhungchu95) [inline app/views/static_pages/about.html.erb]: dùng link_to thay vì a

**Comment by chienpd-0036** [comment]
Use link_to + check Rubocop
Screen Shot 2019-06-28 at 13 29 10

**Comment by chienpd-0036** [comment]
ready

**Comment by chienpd-0036** [comment]
ready

**Pull request merged**